autosklearn/evaluation/train_evaluator.py

```diff
@@ -1037,26 +1097,26 @@ def get_splitter(self, D):
 
 # create closure for evaluating an algorithm
 def eval_holdout(
-        queue,
-        config,
-        backend,
-        resampling_strategy,
-        resampling_strategy_args,
-        metric,
-        seed,
-        num_run,
-        instance,
-        scoring_functions,
-        output_y_hat_optimization,
-        include,
-        exclude,
-        disable_file_output,
-        port,
-        init_params=None,
-        iterative=False,
-        budget=100.0,
-        budget_type=None,
-):
+    queue: multiprocessing.Queue,
+    config: Union[int, Configuration],
+    backend: Backend,
+    resampling_strategy: Union[str, BaseCrossValidator, _RepeatedSplits, BaseShuffleSplit],
+    resampling_strategy_args: Dict[str, Optional[Union[float, int, str]]],
+    metric: Scorer,
+    seed: int,
+    num_run: int,
+    instance: Union[str, bytes],
```

Comment:
This is because of partial fit, where we provide [here](https://github.com/automl/auto-sklearn/blob/5876fb61a905cb6ceff0000a758985dd9b29b333/autosklearn/evaluation/train_evaluator.py#L1208) bytes coming from SMBO

Reply:
Sorry, I think there's something wrong with that link. Where does SMBO provide bytes? Is it when using `json.dumps` to create the instance string? According to [the Python docs](https://docs.python.org/3/library/json.html#json.dumps) this returns a string.

Reply:
This was a suggestion from mypy itself. But if I remove it, there is no error coming from it. I have remove it.